message_parser/src/main/java/cumulus_message_adapter/message_parser/MessageAdapter.java

```diff
@@ -14,7 +14,7 @@
 
 public class MessageAdapter implements IMessageAdapter
 {
-    private static final int MESSAGE_ADAPTER_TIMEOUT = 10; // seconds
+    private static final int MESSAGE_ADAPTER_TIMEOUT = 120; // seconds
```

Comment:
What happens when this times out?  Does it display a message saying that this timed out?

Reply:
It says process has exited. Same message Mike got today which is how i knew it was the timeout.